test/wrapper.test.ts

```diff
@@ -625,4 +626,14 @@ describe('Tests Native Wrapper', () => {
       expect(SentryCapacitor.fetchNativeDeviceContexts).not.toBeCalled();
     });
   });
+
+  describe('closeNativeSdk', () => {
```

Comment:
The current test looks good. Let's add one more for two calls after each other, to ensure this won't throw in future.